python webpage.py -> manubot webpage (#269)
manubot/rootstock: webpage/README.md

> The `deploy.sh` script run during continuous deployment creates the `.ots` files.

Reviewer: ```suggestion
The `deploy.sh` script run during continuous deployment creates the `.ots` files through its `manubot webpage` call.
```

There is a delay before timestamps get confirmed by a Bitcoin block.
Therefore, `.ots` files are initially incomplete and should be upgraded at a later time, so that they no longer rely on the availability of a calendar server to verify.

> `manubot webpage`, which is run during continuous deployment, identifies files matched by `webpage/v/**/*.ots` and attempts to upgrade them.

Reviewer: ```suggestion
The `manubot webpage` call during continuous deployment identifies files matched by `webpage/v/**/*.ots` and attempts to upgrade them.
```
